services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -29,29 +26,22 @@ namespace TNO.Services.FileMonitor;
 public class FileMonitorAction : IngestAction<FileMonitorOptions>
 {
     #region Variables
-    private readonly IHttpRequestClient _httpClient;
     private readonly IKafkaMessenger _kafka;
     private readonly ILogger _logger;
-    private readonly IApiService _api;
-    private readonly IOptions<FileMonitorOptions> _options;
     #endregion
 
     #region Constructors
     /// <summary>
     /// Creates a new instance of a FileMonitorAction, initializes with specified parameters.
     /// </summary>
-    /// <param name="httpClient"></param>
     /// <param name="api"></param>
     /// <param name="kafka"></param>
     /// <param name="options"></param>
     /// <param name="logger"></param>
-    public FileMonitorAction(IHttpRequestClient httpClient, IApiService api, IKafkaMessenger kafka, IOptions<FileMonitorOptions> options, ILogger<FileMonitorAction> logger) : base(api, options)
+    public FileMonitorAction(IApiService api, IKafkaMessenger kafka, IOptions<FileMonitorOptions> options, ILogger<FileMonitorAction> logger) : base(api, options)
```

Comment:
Removed unused parameters.

Reply:
Need to watch linting errors more closely, there were tons of issues in this file.

Reply:
Strange. I received zero linting errors on compile.

Reply:
I received no warnings either. How do you see the warnings? I check the terminal output when I compile and I don't recall seeing any.

Reply:
They show up in the UI.  Perhaps I have a plugin that is relevant, but I believe they show up in the devcontainer too.